src/main/java/seedu/momentum/model/project/Project.java

```diff
@@ -82,6 +82,10 @@ public Description getDescription() {
         return description;
     }
 
+    public boolean hasDescription() {
+        return !description.equals(Description.EMPTY_DESCRIPTION);
+    }
+
```

Comment:
Should this method be in Description instead? It doesn't require anything form project and is only calling a method from description, so I think we should put it together with all the other description code.

Reply:
I agree with Farrell